### ISSUE TITLE
Clarification on setting up Otel using Otel exporter

### ISSUE THREAD

**Issue by cmergenthaler** [spring-projects-experimental/spring-cloud-sleuth-otel#155]
**Describe the bug**
Hey all, 
I am confused on setting up a simple Spring Cloud Sleuth application that exports traces to Otel Collector using Sleuth Otel. I have included all dependencies as stated in your [docs](https://github.com/spring-projects-experimental/spring-cloud-sleuth-otel#quick-start) and I also set the local collector's endpoint in my applications.yaml with `spring.sleuth.otel.exporter.otlp.endpoint: http://collector:4317`. But it seems like this is not enough, after trying a lot of things (incl. #94), I came up also adding the `io.opentelemetry:opentelemetry-exporter-otlp`. That seemed to solve my issue, but I am still wondering if this is really necessary. Please provide some clarification on that.

**Sample**
pom.xml
``` xml
<dependencyManagement>
  <dependencies>
  <!-- Spring Cloud: https://spring.io/projects/spring-cloud -->
      <dependency>
                <groupId>org.springframework.cloud</groupId>
                <artifactId>spring-cloud-dependencies</artifactId>
                <version>2021.0.5</version>
                <type>pom</type>
                <scope>import</scope>
      </dependency>
      <dependency>
                  <groupId>org.springframework.cloud</groupId>
                  <artifactId>spring-cloud-sleuth-otel-dependencies</artifactId>
                  <version>1.1.2</version>
                  <scope>import</scope>
                  <type>pom</type>
      </dependency>
  </dependencies>
</dependencyManagement>

</dependencies>
        <dependency>
            <groupId>org.springframework.cloud</groupId>
            <artifactId>spring-cloud-starter-sleuth</artifactId>
            <exclusions>
                <exclusion>
                    <groupId>org.springframework.cloud</groupId>
                    <artifactId>spring-cloud-sleuth-brave</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <!-- This dependency adds OTel support -->
        <dependency>
            <groupId>org.springframework.cloud</groupId>
            <artifactId>spring-cloud-sleuth-otel-autoconfigure</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-exporter-otlp</artifactId>
        </dependency>
</dependencies>
```

application.yml
```yaml
spring:
    otel:
      config:
        trace-id-ratio-based: 1.0
      exporter:
        otlp:
          endpoint: http://collector:4317
```

docker-compose.yaml
```yaml
version: '3'
services:
 service:
    image: service

  collector:
    image: otel/opentelemetry-collector
    platform: linux
    command: [ "--config=/etc/otel-collector-config.yml" ]
    volumes:
      - ./otel-config.yml:/etc/otel-collector-config.yml
    ports:
      - "4317:4317"
    depends_on:
      - jaeger-service

  jaeger-service:
    image: jaegertracing/all-in-one:latest
    platform: linux
    ports:
      - "16686:16686"
      - "16685:16685"
      - "14250"
```

otel-config.yml
```yaml
receivers:
  otlp:
    protocols:
      http:
      grpc:

processors:
  batch:

exporters:
  logging:
    verbosity: detailed
  jaeger:
    endpoint: jaeger-service:14250
    tls:
      insecure: true

service:
  pipelines:
    traces:
      receivers: [ otlp ]
      processors: [ batch ]
      exporters: [ logging, jaeger ]
```

**Comment by marcingrzejszczak** [comment]
I think you have to set `io.opentelemetry:opentelemetry-exporter-otlp` because you need to use the OTel OTLP exporter. Are you willing to update our docs to add that information?

**Comment by cmergenthaler** [comment]
@marcingrzejszczak thanks, i have created a PR for that

**Comment by cmergenthaler** [comment]
Fixed in #157
